### DISCUSSION TITLE
Update documentation for GitHub Actions `runner` context to include `environment` property

### DISCUSSION THREAD

**Discussion by maskati** [community/community#118413] [Actions]
### Select Topic Area

Bug

### Body

The documentation for the [runner context](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context) does not indicate a new property `environment`:
- Value is `github-hosted` for GitHub hosted runners
- Value is `self-hosted` for self-hosted runners

**Comment by jsoref** [comment]
You'll want to file this in https://github.com/github/docs/issues/new/choose

**Reply by maskati** [reply]
Thanks for the info, I created an [issue](https://github.com/github/docs/issues/32443) in the correct repo.
